sampo/scheduler/genetic/base.py

```diff
@@ -39,10 +37,12 @@ def __init__(self,
                  seed: Optional[float or None] = None,
                  n_cpu: int = 1,
                  weights: list[int] = None,
-                 fitness_constructor: Callable[[Toolbox], FitnessFunction] = TimeFitness,
+                 fitness_function: Callable[[list[Schedule]], list[int]] = TimeFitness().evaluate_from_schedules,
```

Comment:
We should pass `FitnessFunction` constructor here, not it's method;
because in some cases (for example, deadline) we should construct `FitnessFunction` object inside the `build_schedule` method